test/model/compose_test.dart

```diff
@@ -334,6 +335,80 @@ hello
     });
   });
 
+  test('fallbackMarkdownLink', () async {
+    final store = eg.store();
+    final channels = [
+      eg.stream(streamId: 1, name: '`code`'),
+      eg.stream(streamId: 2, name: 'score > 90'),
+      eg.stream(streamId: 3, name: 'A*'),
+      eg.stream(streamId: 4, name: 'R&D'),
+      eg.stream(streamId: 5, name: 'UI [v2]'),
+      eg.stream(streamId: 6, name: r'Save $$'),
+    ];
+    await store.addStreams(channels);
+
+    check(channelTopicFallbackMarkdownLink(store: store,
+        channel: channels[1 - 1]))
+      .equals('[#&#96;code&#96;](#narrow/channel/1-.60code.60)');
```

Comment:
This feels like a helper function that's internal to the implementation of `channelLink`, not an interface meant for the rest of the app to use directly. Better to test against the external interface, then:
```suggestion
    check(channelLink(store: store, channel: channels[1 - 1]))
      .equals('[#&#96;code&#96;](#narrow/channel/1-.60code.60)');
```

… In fact, though, that seems to be covered in the other test case below. So just delete this test?